tasks/scraping.go

```diff
@@ -0,0 +1,47 @@
+package tasks
+
+import (
+	"context"
+
+	"go-google-scraper-challenge/models"
+	"go-google-scraper-challenge/services/scraper"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/task"
+)
+
+type ScrapingTask struct {
+	Tasker *task.Task
+}
+
+func (t *ScrapingTask) Name() string {
+	return "scraping"
+}
+
+func (t *ScrapingTask) GetTasker() *task.Task {
+	if t.Tasker == nil {
+		t.Tasker = task.NewTask(t.Name(), "0 * * * * *", perform)
+	}
+
+	return t.Tasker
+}
+
+func perform(_ context.Context) error {
```

Comment:
Possible to add test for it?

Reply:
@junan I don't think we can test a private function 🤔  and also I want to keep this a private function so only the task can  use it